.github/workflows/windows_ci.yml

```diff
@@ -20,7 +20,9 @@ jobs:
         python-version: [3.7]
         architecture: ['x64']
         onnx-version: ['1.7']
-
+    # To get the tool ort_test_dir_utils from ORT
+    env:
+      PYTHONPATH: .\onnxruntime\tools\python
```

Comment:
tools/python is not an official interface and is subject to change.
If we needs some python lines from this folder, we probably just copy it here. @jcwchen

Reply:
I followed the instruction in the document to include the PYTHONPATH to use this PythonTools: https://github.com/microsoft/onnxruntime/blob/master/tools/python/PythonTools.md
I thought these files are more likely to change and actually it needs two files for now. It is possible that the ORT developers will introduce more files for this ORT utility. In that case, I suppose include the tools/python makes more sense here -- at least it is mentioned in the document and it should be maintained.

Reply:
+ @askhade for the discussion. My concern is these internal tools doesn't have backward compatibility and this model test will be easily broken after ONNXRuntime upgrading, while API in ORT python package are well defined and was promised to be backward compatible.

Reply:
Removed the PYTHONPATH thing and uploaded the ort_test_dir_utils.py files. Thank you for the suggestion @wenbingl